RevenueCat/Scripts/Purchases.cs

```diff
@@ -48,36 +46,36 @@ public partial class Purchases : MonoBehaviour
              "This property is obsolete. " +
              "Use revenueCatAPIKeyApple and revenueCatAPIKeyGoogle instead.")]
     // ReSharper disable once InconsistentNaming
-    public string deprecatedLegacyRevenueCatAPIKey;
+    public string DeprecatedLegacyRevenueCatAPIKey;
 
     [Tooltip("RevenueCat API Key specifically for Apple platforms. Get from https://app.revenuecat.com/")]
     // ReSharper disable once InconsistentNaming
-    public string revenueCatAPIKeyApple;
+    public string RevenueCatAPIKeyApple;
 
     [Tooltip("RevenueCat API Key specifically for Android. Get from https://app.revenuecat.com/")]
     // ReSharper disable once InconsistentNaming
-    public string revenueCatAPIKeyGoogle;
+    public string RevenueCatAPIKeyGoogle;
```

Comment:
turns out that we need to revert these 🤦 I'm doing it on the #114 PR. 
Unity's naming standards for stuff that shows up in the Editor say to use camelCase, as opposed to typical C# would would be Pascal.